pyproject.toml

```diff
@@ -2,7 +2,6 @@
 requires = [
     "setuptools",
     "wheel",
-    "attr",
```

Comment:
why?

Reply:
That's from https://github.com/cosmo-epfl/sklearn-cosmo/pull/16/files